packages/server/routes/api/users.js

```diff
@@ -89,7 +92,43 @@ router.get(
         message: error.message,
       });
     }
-  }
+  },
+);
+
+router.put(
+  "/:userId/new",
```

Comment:
I think we should change this to a POST request. While we are updating the user here, this is specifically for onboarding and a user can only onboard once. We're not allowing users to request this route again so it's not idempotent. [Idempotence in PUT vs POST](https://restfulapi.net/rest-put-vs-post/)

Let's also change this route to `POST /:userId/onboarding` to be more specific instead of just "new".